pkg/keystore/keystoretest/keygen/keygen.go

```diff
@@ -27,9 +27,6 @@ import (
 	"os"
 	"strings"
 	"text/template"
-
-	"github.com/coreos/rkt/Godeps/_workspace/src/golang.org/x/crypto/openpgp"
```

Comment:
These should not be removed. The `github.com/coreos/rkt/Godeps/_workspace/src/` part should be removed from imports instead.